packages/grid/_modules_/grid/hooks/features/useComponents.tsx

```diff
@@ -55,7 +57,7 @@ export const useComponents = (
   );
 
   const components: ApiRefComponentsProperty = React.useMemo(() => {
-    const allComponents = { ...DEFAULT_SLOTS_COMPONENTS, ...componentsProp };
+    const allComponents = { ...DEFAULT_SLOTS_COMPONENTS, ...componentSlotsProp };
```

Comment:
A bit confused here. 
So do you want to render an empty component when the component is falsy or do you want to fall back on the default one?

Reply:
From what I understand, right now it renders an empty components. It would be easier and more consistent for developers if it was rendering the default component.